AmazonWeb1/src/main/java/com/amazon/controller/AuthenticationController.java

```diff
@@ -5,13 +5,30 @@
 import com.amazon.service.Impl2.AuthenticationServiceImpl2;
 import com.amazon.service.impl.AuthenticationServiceImpl;
 
+/**
+ * <p>
+ *     Represents the authentication controller
+ * </p>
+ * @author Roshan
+ * @version 1.0
+ */
 public class AuthenticationController {
 
     private static final AuthenticationController AUTHENTICATION_CONTROLLER = new AuthenticationController();
```

Comment:
I'm using Eagar initialization for singleton design patterns. 
so that instance available from the class initialization to reduce the delay in lazy initialization